modules/installation-aws-about-government-region.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-about-government-region_{context}"]
+= AWS Government and air-gapped regions
+
+{product-title} supports deploying a cluster to AWS Government and other
+air-gapped regions. These hidden regions do not have published {op-system-first}
+AMIs to select, so you must upload a custom AMI that belongs to that region.
+
+Common AWS regions that do not have published {op-system} AMIs include:
+
+* AWS GovCloud, Secret, Top Secret and other secret cloud services
+* Specialized AWS public commercial services
+* Other hidden regions
```

Comment:
cc @katherinedube 

This should be AWS gov cloud only right not other secret, top secret?

Reply:
For 4.6, it should be GovCloud only until we support C2S & SC2S regions.

Reply:
Should I remove everything except GovCloud outlined in that bulleted list? Or are there use cases I missed that should be added?

Reply:
Yes. For now let's only mention GovCloud until we add support for the other regions.